Loading npm modules from inside vue files

I've encounter a problem when attempting to import an npm module from my .vue files. Here is my brunch config:
```javascript
module.exports = {
  files: {
    javascripts: {
      joinTo: {
        'js/bundle.js': /^app\/js|(.|\/)*node_modules/
      }
    },
    stylesheets: {
      joinTo: 'css/bundle.css'
    }
  },
  modules: {
    autoRequire: {
      'js/bundle.js': ['js/main']
    }
  }
};
```

and this is a module:
```javascript
<template lang="html">
  <div class="list-wrapper">
    <ul class="list">
      <li v-for="book in list">{{book.title}}</li>
    </ul>
  </div>
</template>

<script>
  import _ from 'lodash';
  
  export default {
    name: 'BookList',
    data() {
      return {
        list: [
          {
            title: `You Don't Know JS: ES6 & Beyond`,
            author: `Kyle Simpson`,
            year: 2015
          },
          {
            title: `Secrets of the JavaScript Ninja`,
            author: [`John Resig`, `Bear Bibeault`, `Josip Maras`],
            year: 2014
          }
        ]
      }
    },
    computed: {
      newest() {
        return _(this.list).filter((item) => {
          return item.year > 2014;
        }).value();
      }
    }
  }
</script>

<style lang="less">
</style>
```
I compile this stuff and get this error in Chrome console:

`bundle.js:61 Uncaught Error: Cannot find module 'lodash' from 'js/components/book-list.vue'`.

When I looked at bundle.js file I discovered that lodash module has not been included in the bundle. However, if I import lodash module in my main.js file, import in vue component start working and lodash includes in the bundle. It seems like vue-brunch (and vueify under the hood) just compile .vue files and inserts resulting js into bundle without resolving it's dependencies.

I am using brunch@2.10.9, vue-brunch@2.0.1 and vue@2.3.3

--- a/index.js
+++ b/index.js
@@ -54,7 +54,7 @@ class VueBrunch {
 }
 
 VueBrunch.prototype.brunchPlugin = true;
-VueBrunch.prototype.type = 'template';
+VueBrunch.prototype.type = 'javascript';
 VueBrunch.prototype.extension = 'vue';
 
 module.exports = VueBrunch;

